### ISSUE TITLE
Disc Control does not work with ZIP files when running Windows

### ISSUE THREAD

**Issue by PoloniumRain** [schellingb/dosbox-pure#302]
ZIP files which contain discs cannot be inserted into the Windows E: disc drive by using Disc Control. Not sure if this is a bug or just isn't supported, but i'm guessing it's the latter! 

To recreate:

1. Run any zipped game which contains a disc (.ISO or .CUE)
2. Run the installed OS and now the disc will appear in the Windows E: disc drive
3. Go to Disc Control > Eject
4. Go to 'Load New Disc' and choose any other ZIP file that contains a disc
5. Go on 'Insert Disc'
6. The disc wont appear in the Windows E: disc drive

Only unzipped .ISO and .CUE files work with Disc Control. 

It looks like the only way to use ZIP files is to start the core, mount the disc first, then run the installed OS. So any multi-disc games will always need to be unzipped. 

This is also a bit of a problem for me because i have all games installed on C: for a few reasons, but mostly so that i don't have to close the core just to run another game. So i've had to unzip all my Windows games. It's also possible to kinda get around this with no-CD cracks/patches, but then any FMV's and CD-ROM music obviously wont play.

**Comment by ryz** [comment]
Hm, I just installed American McGee's Alice, which requires changing discs at install. I have both discs in the same ZIP file and they're both automatically and correctly loaded at start. At runtime I swap discs via Disc Control > Eject > Current Disc Index from 1 to 2 > Insert Disc.

**Comment by PoloniumRain** [comment]
Yeah, you're right! Sorry. Multiple disc games do actually work. But going to 'Load New Disc' and selecting another ZIP file is what doesn't work. The Windows E: disc drive will just say "The device is not ready" when clicked on.

**Comment by schellingb** [comment]
Yeah this doesn't really work as one would expect.

If you do this outside of Windows during the start menu and look what appears in the D: drive you might realize what's going on. In DOS it will load the content of the ZIP file as the D: drive, not actually any CD images inside of it. So in DOS you can actually load a ZIP and it will emulate a CD-ROM with the content that way. I don't think anyone uses this "feature" but that's all that loading a ZIP via the disc control menu does for now.

It could actually mount the CD if all the ZIP contains was just a single CD image file. But what if there are other files or multiple CD images? Then it becomes tricky...
